src/bindings/fluid_cmd.c

```diff
@@ -20,12 +20,12 @@
 
 #include <glib.h>
 
-#include "fluidsynth_priv.h"
+#include <fluidsynth_priv.h>
```

Comment:
Is this really necessary to be able to override fluidsynth_priv.h with your own header? Couldn't you solve this with a custom include path for the quote-headers, like the -iquote option for gcc?

Reply:
Nope quote includes always looks first one the directory of the file that make the include

Reply:
What keeps you from hard-replacing these files with your own header?

Reply:
Ok I'll revert these changes, an replace these files in my own branch